src/gui/systray.cpp

```diff
@@ -82,8 +82,12 @@ Systray::Systray()
 
 #ifndef Q_OS_MAC
     auto contextMenu = new QMenu();
-    contextMenu->addAction(tr("Open main dialog"), this, &Systray::openMainDialog);
-    contextMenu->addAction(tr("Settings"), this, &Systray::openSettings);
+    if (AccountManager::instance()->accounts().isEmpty()) {
+        contextMenu->addAction(tr("Open Wizard"), this, &Systray::openAccountWizard);
```

Comment:
The word "Wizard" is a bit strange, so we should call it "Add account". Also note only the first word starts capitalized.

(Just saw that the dialog title also says "Nextcloud Connection Wizard" → there we should also simply say "Add account".)